### PR TITLE
[MSHARED-1203] no longer need to shell out to create a symbolic link

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

File: src/test/java/org/apache/maven/shared/model/fileset/util/FileSetUtilsTest.java [hunk 1]
```diff
@@ -269,23 +262,19 @@ void testDeleteDontFollowSymlinksButDeleteThem() throws Exception {
...
-        cli.createArg().setValue(link.getPath());
-
-        int result = cli.execute().waitFor();
-
-        linkFiles.add(link);
-
-        return result == 0;
+        try {
+            Files.createSymbolicLink(link.toPath(), target.toPath());
+            linkFiles.add(link);
```
Comment (slawekjaranowski) [inline src/test/java/org/apache/maven/shared/model/fileset/util/FileSetUtilsTest.java:272]: Do we need to track create links in @TempDir?, should be removed by JUnit

Comment (elharo) [reply src/test/java/org/apache/maven/shared/model/fileset/util/FileSetUtilsTest.java]: good point. done

**Review by slawekjaranowski — APPROVED** [review]

**Review by cstamas — COMMENTED** [review]
Lgtm

**Review by slachiewicz — APPROVED** [review]
Great! One step closer to remove dependency to plexus utils

**Pull request merged**